lib/blacklight/configuration.rb

```diff
@@ -288,15 +276,12 @@ def deep_copy
     end
     alias_method :inheritable_copy, :deep_copy
 
-    ##
     # Get a view configuration for the given view type
     # including default values from the index configuration
+    # @param [Symbol,#to_sym] view_type
     def view_config(view_type)
-      if view_type == :show
-        self.index.merge self.show
-      else
-        self.index.merge view.fetch(view_type, {})
-      end
+      view_type = view_type.to_sym unless view_type.is_a? Symbol
```

Comment:
If we do this here, then aren't necessitated to do this everywhere that may want to use `view_type`? Why not just pop this down as @cbeer suggests?